src/app/core.ts

```diff
@@ -70,13 +70,26 @@ export const initDragDrop = (): Promise<boolean> => {
  * will be associated with this ID. It is recommended to use a unique ID to keep transfer
  * information private from other websites.
  *
+ * @param userSession whether or not to use multiple user sessions (defaults to false). This is
+ * useful to handle multiple user sessions in the same device.
+ *
  * @returns a promise that resolves if IBM Aspera Desktop is running properly or
  * rejects if unable to connect
  */
-export const initDesktop = (appId?: string): Promise<any> => {
+export const initDesktop = (appId?: string, userSession?: boolean): Promise<any> => {
```

Comment:
Maybe better if we change the signature of this function to take in an options object instead with `appId` and `userSession` as keys in the object. Gives us more flexibility to change the signature in the future.

Also `userSession` to me is a little unclear or seems weird that it actually enables the flow that supports multiple users. `supportMultipleUsers` or something like that would be more clear?

Reply:
Done!